### PR TITLE
EPMRPP-62695 - make decision modal add tooltip

### PR DESCRIPTION
**Pull request by chivekrodis**

### CODE REVIEW

File: app/localization/translated/be.json [hunk 1]
```diff
@@ -1075,17 +1075,20 @@
...
   "MakeDecisionModal.allLoadedTIFromHistoryLine": "Усе загружаныя TI з гісторыі",
   "MakeDecisionModal.apply": "Прымяніць",
-  "MakeDecisionModal.applyImmediately": "Ужыць неадкладна",
+  "MakeDecisionModal.applyAndContinue": "Прымяніць і працягваць",
   "MakeDecisionModal.applyDefectFor": "Прымяніць дэфект для",
+  "MakeDecisionModal.applyImmediately": "Ужыць неадкладна",
   "MakeDecisionModal.applyToItems": "Прымяніць для {itemsCount} элементаў",
   "MakeDecisionModal.applyToSimilarItems": "Вы таксама можаце ўжыць гэтае рашэнне да іншых аналагічных элементаў у",
-  "MakeDecisionModal.applyAndContinue": "Прымяніць і працягваць",
+  "MakeDecisionModal.applyToTheItem": "Apply to the Item",
```
Comment (AmsterGet) [inline app/localization/translated/be.json:1083]: Should it be translated?

File: app/localization/translated/ru.json [hunk 2]
```diff
@@ -1012,7 +1012,7 @@
   "LinkIssueModal.issueLinkPlaceholder": "Введите ссылку на ошибку",
   "LinkIssueModal.link": "Прикрепить",
   "LinkIssueModal.linkIssueFailed": "Не удалось прикрепить ошибку",
-  "LinkIssueModal.postIssueForTheTest": "Прикрепить ошибку для теста {launchNumber}",
+  "LinkIssueModal.linkIssueForTheTest": "Link Issue for the test {launchNumber}",
```
Comment (AmsterGet) [inline app/localization/translated/ru.json:1015]: Same here

File: app/localization/translated/uk.json [hunk 3]
```diff
@@ -1075,17 +1075,20 @@
...
   "MakeDecisionModal.allLoadedTIFromHistoryLine": "Всі завантажені TI з історії",
   "MakeDecisionModal.apply": "Застосувати",
-  "MakeDecisionModal.applyImmediately": "Застосувати негайно",
+  "MakeDecisionModal.applyAndContinue": "Застосувати та продовжити",
   "MakeDecisionModal.applyDefectFor": "Застосувати дефект для",
+  "MakeDecisionModal.applyImmediately": "Застосувати негайно",
   "MakeDecisionModal.applyToItems": "Примінити для {itemsCount} елементів",
   "MakeDecisionModal.applyToSimilarItems": "Ви також можете застосувати це рішення до інших подібних елементів у",
-  "MakeDecisionModal.applyAndContinue": "Застосувати та продовжити",
+  "MakeDecisionModal.applyToTheItem": "Apply to the Item",
```
Comment (AmsterGet) [inline app/localization/translated/uk.json:1083]: Same here

File: app/src/components/inputs/inputRadio/inputRadio.jsx [hunk 4]
```diff
@@ -32,6 +32,7 @@ export const InputRadio = ({
   onBlur,
   mobileDisabled,
   inputClassNames,
+  tooltip,
```
Comment (AmsterGet) [inline app/src/components/inputs/inputRadio/inputRadio.jsx:35]: Why not just `title`? Seems that it is default browser titile.

File: app/src/components/inputs/inputRadioGroup/inputRadioGroup.jsx [hunk 5]
```diff
@@ -36,6 +36,10 @@ export class InputRadioGroup extends PureComponent {
           id: PropTypes.string.isRequired,
           defaultMessage: PropTypes.string.isRequired,
         }),
+        tooltip: PropTypes.shape({
```
Comment (AmsterGet) [inline app/src/components/inputs/inputRadioGroup/inputRadioGroup.jsx:39]: Same here

File: app/src/components/main/modal/darkModalLayout/modalHeader/modalHeader.scss [hunk 6]
```diff
@@ -53,6 +53,8 @@
   cursor: pointer;
 
   svg {
+    width: 30px;
+    height: 30px;
```
Comment (AmsterGet) [inline app/src/components/main/modal/darkModalLayout/modalHeader/modalHeader.scss:57]: Is size inside of `.close-icon`  still needed?

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/localization/translated/be.json (hunk 1 above)
Comment (chivekrodis) [reply app/localization/translated/be.json:1083]: unused property. I have remove it

File: app/localization/translated/uk.json (hunk 3 above)
Comment (chivekrodis) [reply app/localization/translated/uk.json:1083]: unused property. I have removed it

File: app/src/components/main/modal/darkModalLayout/modalHeader/modalHeader.scss (hunk 6 above)
Comment (chivekrodis) [reply app/src/components/main/modal/darkModalLayout/modalHeader/modalHeader.scss:57]: yes in safari this image doesn't show without size. As a different solution move it in the .svg file

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**